ui/app/components/auth/form/base.ts

```diff
@@ -39,7 +40,9 @@ export default class AuthBase extends Component<Args> {
     const data: Record<string, FormDataEntryValue | null> = {};
 
     for (const key of formData.keys()) {
-      data[key] = formData.get(key);
+      const value = formData.get(key);
+      // strip leading or trailing slashes from path for consistency
+      data[key] = key === 'path' ? sanitizePath(value) : value;
```

Comment:
this may end up moving once the component is wired up and [auth logic](https://github.com/hashicorp/vault/blob/349e449d49fef4fcc836d29b84ec7c1336bdb9ca/ui/app/adapters/cluster.js#L178-L196) is removed from the clusterAdapter 

for now, serialization is happening here for consistency so it's not sprinkled all over the place like it is now 🥲